aquarius/ddo_checker/shacl_schemas/v4/remote_4.2.0.ttl

```diff
@@ -0,0 +1,428 @@
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:type;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:label;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:required ;
+        sh:datatype xsd:boolean ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:description ;
+        sh:datatype xsd:string ;
+        sh:pattern "^((.|\n)*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 65536;
+    ] ;
+    sh:property [
+        sh:path schema:default;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:options;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxLength 256;
+    ] ;.
+
+
+schema:ContainerShape
+    sh:targetClass schema:Container ;
+    sh:property [
+        sh:path schema:entrypoint ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:image ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:tag ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:checksum ;
+        sh:datatype xsd:string ;
+        sh:pattern "^sha256\\:(.*)$" ;
+        sh:maxLength 71;
+        sh:minLength 71;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;.
+
+
+schema:ServiceShape
+    sh:targetClass schema:Service ;
+    sh:property [
+        sh:path schema:id ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:type ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:name ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:description ;
+        sh:datatype xsd:string ;
+        sh:pattern "^((.|\n)*)$" ;
+        sh:maxCount 1;
+        sh:maxLength 65536;
```

Comment:
65536 characters seems very long, something like 10000 words which would be 40+ pages. Is there any benefit to having it this high? In the market the max characters for the description is set to 5000.  This could probably also benefit from having a `minLength`, in the market the minimum is 10

Reply:
I added the minLength.
For maxLength, I wanted to err on the side of tolerance, but I'm open to suggestions. @alexcos20 do you think 5000 is enough for asset and service description? I'd like to have the same restrictions on both, if applicable.

Reply:
let's have it 7000.  should be more than enough, and non blocking on our side